lib/data/domain/books_model.dart

```diff
@@ -27,3 +30,17 @@ class BookModel {
     );
   }
 }
+
+class UserPreferences {
```

Comment:
This offers similar functions to `utils.dart`; should the functions not be grouped together?